### PR TITLE
fix: allow either specifiers or path

### LINKED ISSUE #6: Dead filter
https://github.com/jdvivar/eleventy-plugin-add-web-component-definitions/blob/3cdf1df4036e49fc41a9e78e14a898a5a830a6e6/add-web-component-definitions.js#L66

...is now dead (unless path is a null string, which could/should be another throw Error), and the line can be deleted, because of this improvement...

https://github.com/jdvivar/eleventy-plugin-add-web-component-definitions/blob/3cdf1df4036e49fc41a9e78e14a898a5a830a6e6/add-web-component-definitions.js#L61

### PR DESCRIPTION
**Pull request by bennypowers**
Closes #6

### CODE REVIEW

**Comment by bennypowers** [comment]
added some tests as well

**Comment by gdbaldw** [comment]
I understand it your code. I humbly suggest considering `path: ""`, which may yield the result you need. If acceptable, we could simply keep the Boolean filter, update the Documentation regarding `path: ""`, and close #6. The benefit is increased configuration control, having the catchall `path` for anything not specified, or the option of only those specified when `path: ""`.

File: add-web-component-definitions.js [hunk 1]
```diff
@@ -16,11 +16,19 @@ const addChild = (body, child, position) => {
...
+  if (!!options.path && !!options.specifiers) {
+    throw new Error('You may configure a path function or import specifiers, but not both');
+  }
+
+  if (!options.path && !options.specifiers) {
+    options = Object.assign({
+      path(tag) { return `/js/components/${tag}/${tag}.js` },
+    }, options);
+  }
+
```
Comment (jdvivar) [inline add-web-component-definitions.js:28]: This looks much better

**Review by jdvivar — APPROVED** [review]
I'll publish a new version right after merging this, great work

**Comment by jdvivar** [comment]
> I understand it your code. I humbly suggest considering `path: ""`, which may yield the result you need now, without changes. If acceptable, we could keep the Boolean filter, update the Documentation regarding `path: ""`, and close #6. The benefit is increased configuration control, with either a catchall `path` for anything not specified, or no catchall and only those specified when `path: ""`.
> 
> A related suggestion is to allow `specifiers` to accept a glob, so that the specification `"*": /js/components/${tag}/${tag}.js` becomes equivalent to the current `path` option. Would depreciate `path`.

Sorry, I read the code and didn't read your suggestion before accepting the merge. I'm going to think about the functionality you propose and get back with a new PR for you @gdbaldw

**Pull request merged**